CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ Documentation for rocPRIM is available at
 ## Unreleased rocPRIM-3.3.0 for ROCm 6.3.0
 
 ### Additions
-
+* Add --test smoke option in retest.py. It will run a subset of tests such that the total test time is in 5 minutes. Use python3 ./rtest.py --test smoke or python3 ./rtest.py -t smoke to execute smoke test.
```

Comment:
Thanks for the consistency fix.   Sorry missed what expect is a typo "retest.py" you mean "rtest.py" right?

Reply:
Thanks for pointing this out! Its been updated